paper-tabs.html

```diff
@@ -103,6 +103,12 @@
       -webkit-tap-highlight-color: rgba(0,0,0,0);
 
       @apply(--paper-tabs);
+
+      display: flex;
+    }
+
+    :host-context([dir=rtl]) {
+      flex-direction: row-reverse;
```

Comment:
Should this maybe use `@apply(--layout-horizontal-reverse)`? i think you still need `-ms-flex` and `-webkit-flex` prefixes otherwise.

Same, above, there's browser prefixes for flex, i think? (and we have `@apply(--layout)`)